modules/ROOT/pages/query-unified-schema.adoc

```diff
@@ -1,15 +1,15 @@
-= Querying the Unified Schema
+= Running Queries and Mutations
 
-The unified schema resolves into a single GraphQL endpoint. To query the unified schema, you first request access (if applicable) and then write and run a GraphQL query in the query editor.
+The unified schema resolves into a single GraphQL endpoint. To run queries or mutations against the unified schema, you first request access (if applicable) and then write and run a GraphQL query in the query editor.
 
 == Prerequisites
 
-* You must have xref:permissions.adoc[the Consume, Contribute, or Admin permission] to use request access and run queries.
+* You must have xref:permissions.adoc[the Consume, Contribute, or Admin permission] to use request access and run queries and mutations.
```

Comment:
```suggestion
* You must have xref:permissions.adoc[the Consume, Contribute, or Admin permission]. 
```
We don't _have_ to tell _why_ in a prereq. list item. Your choice. If you keep...:
```suggestion
* You must have xref:permissions.adoc[the Consume, Contribute, or Admin permission] to run queries and mutations.
```